src/notifications/src/index.ts

```diff
@@ -0,0 +1,45 @@
+let nodemailer = require('nodemailer');
+const Email = require('email-templates');
+
+require('dotenv').config({ path: '../keys.env'});

```

Comment:
does importing work? e.g. ```import nodemailer from 'nodemailer'```
There might be typings that exist for these npm packages: https://www.npmjs.com/package/@types/nodemailer

Reply:
From my experience, importing always gives errors so I always use require in favor of import.

Reply:
I'm sure that these errors are from type errors and you have to install those type package to ensure that all the types are working.  ```require``` disables types in TypeScript and it becomes more like JavaScript instead.